README.md

```diff
@@ -58,6 +78,29 @@ $( selector ).imageLightbox({
 });
 ````
 
+| Property | Default | Type | Description |
+| -------- | ------- | ---- | ----------- |
+|selector|'a[data-imagelightbox]'|string||
+|id|'imagelightbox'|string|The id of the preview image/video element|
+|allowTypes|'png&#124;jpg&#124;jpeg&#124;gif'|string|use empty string to allow any file type|
+|animationSpeed|250|integer||
+|activity|false|bool|show activity indicator|
+|arrows|false|bool|show left/right arrows|
+|button|false|bool|show close button|
+|caption|false|bool|show captions|
+|enableKeyboard|true|bool|enable keyboard shortcuts (arrows Left/Right and Esc)|
+|history|false|bool|enable image permalinks and history|
+|fullscreen|false|bool|enable fullscreen (enter/return key)|
+|gutter|10|integer|window height less height of image as a percentage|
+|offsetY|0|integer|vertical offset in terms of gutter|
+|navigation|false|bool|show navigation|
+|overlay|false|bool|display the lightbox as an overlay|
+|preloadNext|true|bool|silently preload the next image|
+|quitOnEnd|false|bool|quit after viewing the last image|
+|quitOnImgClick|false|bool|quit when the viewed image is clicked|
+|quitOnDocClick|true|bool|quit when anything but the viewed image is clicked|
+|quitOnEscKey|true|bool|quit when Esc key is pressed|
+
```

Comment:
This should probably just replace the code snippet above - just reference there that the constructor takes options as a parameter and then list them in this table?